### DISCUSSION TITLE
1.1

### DISCUSSION THREAD

**Discussion by MattStypa** [MattStypa/DOS-Director#5] [Releases]
> [!NOTE]
>  ### 📒 [Documentation](https://github.com/MattStypa/DOS-Director/blob/1.1/README.md)

# What's new
- Much better memory management. The number of items in a menu is now over 30,000
- The cursor position will now be saved when navigating between menus
- Nested meta tags. See the documentation for more information about the `{before}` and `{after}` tags

<hr /><em>This discussion was created from the release <a href='https://github.com/MattStypa/DOS-Director/releases/tag/1.1'>1.1</a>.</em>

**Comment by d0lphinius** [comment]
Suggestion: add image display for games

![image](https://github.com/user-attachments/assets/2a7afdb5-5408-4572-99f0-713ec435f998)

**Reply by MattStypa** [reply]
@d0lphinius 
I have been thinking about this quite a bit.

The current version of the DOS Director is purely an ASCII based interface making images not really something that would work.

Although, I am hoping to eventually do a GUI based version as DOS Director 2. This would make images possible.

Currently, I don't have a good plan on how to do it. The config file would have to become a bit more complex. I have some ideas.
